src/main/java/com/microsoft/graph/Requests/IBaseClient.java

```diff
@@ -27,5 +27,5 @@ public interface IBaseClient {
      *
      * @return the BatchRequestBuilder instance
      */
-    public BatchRequestBuilder getBatchRequestBuilder();
+    //public BatchRequestBuilder getBatchRequestBuilder();
```

Comment:
unomment

Reply:
commented out while I refactor their implementation. Temporary only while we are in preview.